src/test/com/aca/homework/week9/invoice/service/InvoiceServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package com.aca.homework.week9.invoice.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+
+class InvoiceServiceTest {
+    private InvoiceService testSubject;
+
+    @BeforeEach
+    public void setUp() throws FileNotFoundException {
+        testSubject = new InvoiceService();
+        testSubject.load("invoices.txt");
```

Comment:
let's create a InvoiceREader implementation and do not read from a real text file